students.ps1

```diff
@@ -145,10 +149,10 @@ try {
             }
             $leerling.Contracts.add($primaryContract)
             # klassen
-            $klassenResult = Invoke-PresentisRestMethod -Uri "$BaseUrl/klassen?schoollocatie=$($schoollocatie.schoollocatieid)&" -Headers  @{Authorization = "Bearer $($responseToken.token)"}
+            $klassenResult = Invoke-PresentisRestMethod -Uri "$BaseUrl/klassen?schoollocatie=$($schoollocatie.schoollocatieid)&" -Headers  @{Authorization = "Bearer $($responseToken.acces_token)"} -ResultProperty "klassen"
```

Comment:
Shouldn't this be `access_token`? or is it misspelled within Presentis?
also apples to lines: 127, 130, 134, 155 and 177.

And maybe, Also update the _person.ps1_ accordingly with correct values.

Reply:
I've checked this once again and "acces_token" is the right one. Furthermore this is applied to the persons.ps1 as well